src/main/java/eu/darkbot/utils/KekkaPlayerProxyServer.java

```diff
@@ -0,0 +1,99 @@
+package eu.darkbot.utils;
+
+import com.github.manolo8.darkbot.core.api.GameAPI;
+
+import javax.net.ssl.SSLSocketFactory;
+import java.io.*;
+import java.net.*;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class KekkaPlayerProxyServer {
+
+    private static final ExecutorService THREAD_POOL = Executors.newFixedThreadPool(8);
+
+    public KekkaPlayerProxyServer(GameAPI.Handler handler, int port) throws IOException {
```

Comment:
What is really the point of an object in this case? the object is stateless and has no methods except the constructor!!

Consider simply making this a static method instead:
`KekkaPlayerProxyServer.runProxy(handler, port)`